cms/urls.py

```
@@ -1,8 +1,16 @@
 from django.conf import settings
 from django.contrib import admin
+from django.http import HttpResponse
 from django.urls import include, path
+from prometheus_client import generate_latest
+
+
+def metrics_view(request):
```

Ruff (0.15.4)

[warning] Unused function argument: `request`

(ARG001)